### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MatthewCaseres**
Some stuff I noticed in the README that needed updating.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,10 +229,9 @@ We welcome PRs! See the [CONTRIBUTING](CONTRIBUTING.md) file.
 We'd love to hear from and work with early adopters to shape our designs. Please reach out by raising an issue if you're
 interested in using this tooling for your project.
 
-## Future Plans
+<!-- ## Future Plans -->
```
Comment (msaroufim) [inline README.md:232]: let's remove the comments?

Comment (MatthewCaseres) [reply README.md]: done

**Review by msaroufim — APPROVED** [review]

**Comment by MatthewCaseres** [comment]
should we get the merge?

**Review by NivekT — APPROVED** [review]
Our team should update the "Future Plans" section soon, but this PR looks good and thanks for spotting these!

**Pull request closed without merging**